src/lib/viewers/media/MediaControls.js

```diff
@@ -762,6 +769,22 @@ class MediaControls extends EventEmitter {
         status.getPromise().then(this.setFilmstrip);
     }
 
+    /**
+     * Called when user moves mouse over the scrubber
+     *
+     * @param {functon} cb the callback function to be executed
```

Comment:
Nitpick - `@param {Function} cb - The callback function to be executed`

Note capital `Function` (we typically capitalize `Object`, `Function`, and `Array` and lowercase `string`, `boolean`, `number` in JSDoc) and dash between variable name and description

Reply:
Good catch, i didnt even manage to spell function right!